src/model/Policy.lua

```diff
@@ -172,6 +172,23 @@ function Policy:hasPolicy(sec, ptype, rule)
     return false
 end
 
+--[[
+     * hasPolicies determines whether a model has any of the specified policies.
+     *
+     * @param sec the section, "p" or "g".
+     * @param ptype the policy type, "p", "p2", .. or "g", "g2", ..
+     * @param rules the policies rules.
+     * @return whether one is found.
+]]
+function Policy:hasPolicies(sec, ptype, rules)
+    for i = 1, #rules do
+        if self:hasPolicy(sec, ptype, rules[i]) then
+            return true
+		end
+    end
+    return false
+end
+
```

Comment:
@Edmond-J-A Format your code and don’t forget to add unit tests .

Reply:
Got it，unit tests has been added.